api/src/services/taxonomy-service.ts

```diff
@@ -34,6 +34,18 @@ export class TaxonomyService {
     });
   };
 
+  async getTaxonomyFromIds(ids: number[]) {
```

Comment:
Do we need this function as we have a very similar function `getSpeciesFromIds`?

Reply:
The main difference is that the EML file breaks the taxonomy down into its parts, so I can't have 1 giant string.  I need the english_name and kingdom, etc, all separately.  Might be other ways to re-arrange this code, but that was just the easiest option for now.

Reply:
I can make one call the other, and just include the string formatting in one of them so the query isn't duplicated.